pkg/api/v1/endpoints/util.go

```diff
@@ -23,7 +23,7 @@ import (
 	"hash"
 	"sort"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
we don't need this change either

Reply:
this is picked up by default on auto save. not sure why.

Reply:
removed.